install_config/aggregate_logging.adoc

```diff
@@ -102,309 +90,418 @@ deployed. To control component placement, specify node selectors per component t
+when `openshift_logging_use_ops` is set to `true`.
 
-[[deploying-the-efk-stack]]
-== Deploying the EFK Stack
+|`openshift_logging_es_ops_pvc_size`
+|Equivalent to `openshift_logging_es_pvc_size` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
 
-The EFK stack is deployed using a
-xref:../dev_guide/templates.adoc#dev-guide-templates[template] to
-create a deployer pod that reads the deployment parameters and manages
-the deployment.
+|`openshift_logging_es_ops_pvc_prefix`
+|Equivalent to `openshift_logging_es_pvc_prefix` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
 
-Run the deployer, optionally specifying parameters (described in the table
-below), for example:
+|`openshift_logging_es_ops_recover_after_time`
+|Equivalent to `openshift_logging_es_recovery_after_time` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
 
-====
-Without template parameters:
+|`openshift_logging_es_ops_storage_group`
+|Equivalent to `openshift_logging_es_storage_group` for Ops cluster
+when `openshift_logging_use_ops` is set to `true`.
 
-----
-$ oc new-app logging-deployer-template
-----
+|`openshift_logging_es_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Elasticsearch instances. This can be used to place
+these instances on nodes reserved or optimized for running them.
+For example, the selector could be `node-type=infrastructure`. At least
+one active node must have this label before Elasticsearch will deploy.
 
-With parameters:
+|`openshift_logging_kibana_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Kibana instances.
 
-----
-$ oc new-app logging-deployer-template \
-             --param IMAGE_VERSION=<tag> \
-             --param MODE=install
-----
+|`openshift_logging_curator_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Curator instances.
+|===
 
-Replace `<tag>` with `{latest-tag}` for the latest version.
+//. Create a xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret] to provide
+//security-related files to the deployer. Providing the secret is optional, and
+//the objects will be randomly generated if not supplied.
+//+
+//You can supply the following files when creating a new secret, for example:
+//+
+//----
+//$ oc create secret generic logging-deployer \
+//    --from-file kibana.crt=/path/to/cert \
+//    --from-file kibana.key=/path/to/key
+//----
+//
+//[cols="3,7",options="header"]
+//|===
+//|File Name
+//|Description
+//
+//|*_openshift_logging_kibana_cert_*
+//|A browser-facing certificate for the Kibana server.
+//
+//|*_openshift_logging_kibana_key_*
+//|A key to be used with the Kibana certificate.
+//
+//|*_openshift_logging_kibana_ops_cert_*
+//|A browser-facing certificate for the Ops Kibana server.
+//
+//|*_openshift_logging_kibana_ops_key_*
+//|A key to be used with the Ops Kibana certificate.
+//
```

Comment:
I see the followings:
openshift_logging_kibana_ops_ca is :
The absolute path on the control node to the CA file to use for the public facing ops kibana certs

I think it's a little improper, customer will think it is CA path, actually it is CA file, so I think it should be changed to the following: The CA file on the control node used for the public facing ops kibana certs